### PR TITLE
docs: add changelog 4.21.1

### PR DESCRIPTION
**Pull request by vagusX**
## 4.21.1

`2022-06-13`

- 🐞 修复 Image 中的 `getContainer` 属性没有读取 ConfigProvider 中的设置。[#36002](https://github.com/ant-design/ant-design/pull/36002) [@robothot](https://github.com/robothot)
- 🐞 修复 Button 有 `href` 时 `disabled` 属性不生效的问题 [#35952](https://github.com/ant-design/ant-design/issues/35952)。[#35975](https://github.com/ant-design/ant-design/pull/35975) [@MuxinFeng](https://github.com/MuxinFeng)
- 🐞 修复 Upload 组件动画闪烁的问题。[#35943](https://github.com/ant-design/ant-design/pull/35943)
- 🐞 修复 less 色彩算法，使其和 `@ant-design/colors` 保持一致。[#35954](https://github.com/ant-design/ant-design/pull/35954) [@christian-lechner](https://github.com/christian-lechner)
- 💄 Form.Item 中的 Modal 或 Drawer 组件包含的控件去除 `status` 等受 Form 影响的样式。[#35849](https://github.com/ant-design/ant-design/pull/35849)
- TypeScript
  - 🤖 修复 typescript 中的 `DropdownProps` 中的 `autoFocus` 属性定义。[#35990](https://github.com/ant-design/ant-design/pull/35990) [@robothot](https://github.com/robothot)
  - 🤖 修复 Menu 中 `MenuItemGroupType` 的类型定义。[#35790](https://github.com/ant-design/ant-design/pull/35790) [@MasaoBlue](https://github.com/MasaoBlue)
  - 🤖 修复 Carousel 在 React 18 下的 TS 定义问题。[#35959](https://github.com/ant-design/ant-design/pull/35959)
  - 🤖 修复 Dropdown TS 类型定义中缺少分号的问题。[#36008](https://github.com/ant-design/ant-design/pull/36008) [@robothot](https://github.com/robothot)
- 🌐 国际化
  - 🇮🇹 修复 `Table.cancelSort` 的意大利语翻译。[#35970](https://github.com/ant-design/ant-design/pull/35970) [@gariggio](https://github.com/gariggio)

----------------- 

## 4.21.1

`2022-06-13`

- 🐞 Fixed Image the `getContainer` property not reading from ConfigProvider. [#36002](https://github.com/ant-design/ant-design/pull/36002) [@robothot](https://github.com/robothot)
- 🐞 Fixed Button issue [#35952](https://github.com/ant-design/ant-design/issues/35952) where the `disabled` attribute does not take effect with `href`. [#35975](https://github.com/ant-design/ant-design/pull/35975) [@MuxinFeng](https://github.com/MuxinFeng)
- 🐞 Fix less color palette algorithm according to `@ant-design/colors`. [#35954](https://github.com/ant-design/ant-design/pull/35954) [@christian-lechner](https://github.com/christian-lechner)
- 🐞 Fix Upload image flickering. [#35943](https://github.com/ant-design/ant-design/pull/35943)
- 💄 Remove styles from Form such as `status` for children of Modal and Drawer. [#35849](https://github.com/ant-design/ant-design/pull/35849)
- TypeScript
  - 🤖 Fix `autoFocus` property definition in `DropdownProps` in typescript. [#35990](https://github.com/ant-design/ant-design/pull/35990) [@robothot](https://github.com/robothot)
  - 🤖 Fix type definition of `MenuItemGroupType` in Menu. [#35790](https://github.com/ant-design/ant-design/pull/35790) [@MasaoBlue](https://github.com/MasaoBlue)
  - 🤖 Fix Carousel TS definition in React 18. [#35959](https://github.com/ant-design/ant-design/pull/35959)
  - 🤖 Fix missing semicolon in Dropdown TS type definition. [#36008](https://github.com/ant-design/ant-design/pull/36008) [@robothot](https://github.com/robothot)
- 🌐 Localization
  - 🇮🇹 Fix italian translation for `Table.cancelSort` key. [#35970](https://github.com/ant-design/ant-design/pull/35970) [@gariggio](https://github.com/gariggio)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,26 @@ timeline: true
 
 ---
 
+## 4.21.1
+
+`2022-06-13`
+
+- 🐞 Fixed the `getContainer` property in Image not reading the settings in ConfigProvider. [#36002](https://github.com/ant-design/ant-design/pull/36002) [@robothot](https://github.com/robothot)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:22]: `Fix Image...` 组件名字都挪到前面~~

File: CHANGELOG.en-US.md [hunk 2]
```diff
@@ -15,6 +15,26 @@ timeline: true
...
+- 🐞 Fixed the issue [#35952](https://github.com/ant-design/ant-design/issues/35952) where the `disabled` attribute does not take effect when a Button has `href`. [#35975](https://github.com/ant-design/ant-design/pull/35975) [@MuxinFeng](https://github.com/MuxinFeng)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:23]: 同上，挪到前面~~

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -15,6 +15,26 @@ timeline: true
...
+- 🐞 Fix less color palette algorithm accord to `@ant-design/colors`. [#35954](https://github.com/ant-design/ant-design/pull/35954) [@christian-lechner](https://github.com/christian-lechner)
+- 🐞 Fix `animation-timing-function` in Upload. [#35943](https://github.com/ant-design/ant-design/pull/35943)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:25]: > 用面向开发者的角度和叙述方式撰写 CHANGELOG，不描述修复细节，描述问题和对开发者的影响；描述用户第一现场的问题，而非你的解决方式。

@MadCcc  重写一下这个 changlog

ref: [WIKI](https://github.com/ant-design/ant-design/wiki/%E8%BD%AE%E5%80%BC%E8%A7%84%E5%88%99%E5%92%8C%E7%89%88%E6%9C%AC%E5%8F%91%E5%B8%83%E6%B5%81%E7%A8%8B#2-%E7%BC%96%E8%BE%91%E6%9B%B4%E6%96%B0%E6%97%A5%E5%BF%97%E4%BB%A5%E5%8F%8A%E5%8D%87%E7%BA%A7%E7%89%88%E6%9C%AC%E5%8F%B7)

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -15,6 +15,26 @@ timeline: true
...
+- 💄 Remove styles from Form such as `status` for children of Modal and Drawer. [#35849](https://github.com/ant-design/ant-design/pull/35849)
+- TypeScript
+  - 🤖 Fix `autoFocus` property definition in `DropdownProps` in typescript. [#35990](https://github.com/ant-design/ant-design/pull/35990) [@robothot](https://github.com/robothot)
+  - 🤖 Fix type definition of `MenuItemGroupType` in Menu. [#35790](https://github.com/ant-design/ant-design/pull/35790) [@MasaoBlue](https://github.com/MasaoBlue)
+  - 🤖 Fix Carousel TS definition in React 18. [#35959](https://github.com/ant-design/ant-design/pull/35959)
+  - 🤖 Fix missing semicolon in Dropdown TS type definition. [#36008](https://github.com/ant-design/ant-design/pull/36008) [@robothot](https://github.com/robothot)
+- Docs
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:32]: docs 的更新可以不用写

File: CHANGELOG.en-US.md (hunk 3 above)
Comment (MadCcc) [reply CHANGELOG.en-US.md]: done

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,23 @@ timeline: true
...
+
+`2022-06-13`
+
+- 🐞 修复 Image 中的 `getContainer` 属性没有读取 ConfigProvider 中的设置。[#36002](https://github.com/ant-design/ant-design/pull/36002) [@robothot](https://github.com/robothot)
+- 🐞 修复 Button 有 `href` 时 `disabled` 属性不生效的问题 [#35952](https://github.com/ant-design/ant-design/issues/35952)。[#35975](https://github.com/ant-design/ant-design/pull/35975) [@MuxinFeng](https://github.com/MuxinFeng)
+- 🐞 修复 Upload 组件动画闪烁的问题。[#35943](https://github.com/ant-design/ant-design/pull/35943)
+- 🐞 修复 less 色彩算法，使其和 `@ant-design/colors` 保持一致。[#35954](https://github.com/ant-design/ant-design/pull/35954) [@christian-lechner](https://github.com/christian-lechner)
+- 💄 Form.Item 中的 Modal 或 Drawer 组件包含的控件去除 `status` 等受 Form 影响的样式。[#35849](https://github.com/ant-design/ant-design/pull/35849)
+- TypeScript
+  - 🤖 修复 typescript 中的 `DropdownProps` 中的 `autoFocus` 属性定义。[#35990](https://github.com/ant-design/ant-design/pull/35990) [@robothot](https://github.com/robothot)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:28]:
```suggestion
  - 🤖 修复 Dropdown `autoFocus` 属性定义。[#35990](https://github.com/ant-design/ant-design/pull/35990) [@robothot](https://github.com/robothot)
```

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -15,6 +15,23 @@ timeline: true
...
+  - 🤖 修复 Menu 中 `MenuItemGroupType` 的类型定义。[#35790](https://github.com/ant-design/ant-design/pull/35790) [@MasaoBlue](https://github.com/MasaoBlue)
+  - 🤖 修复 Carousel 在 React 18 下的 TS 定义问题。[#35959](https://github.com/ant-design/ant-design/pull/35959)
+  - 🤖 修复 Dropdown TS 类型定义中缺少分号的问题。[#36008](https://github.com/ant-design/ant-design/pull/36008) [@robothot](https://github.com/robothot)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:31]: 这个不用写，因为是之前的 PR 引入的。

File: CHANGELOG.zh-CN.md [hunk 7]
```diff
@@ -15,6 +15,23 @@ timeline: true
 
 ---
 
+## 4.21.1
+
+`2022-06-13`
+
+- 🐞 修复 Image 中的 `getContainer` 属性没有读取 ConfigProvider 中的设置。[#36002](https://github.com/ant-design/ant-design/pull/36002) [@robothot](https://github.com/robothot)
+- 🐞 修复 Button 有 `href` 时 `disabled` 属性不生效的问题 [#35952](https://github.com/ant-design/ant-design/issues/35952)。[#35975](https://github.com/ant-design/ant-design/pull/35975) [@MuxinFeng](https://github.com/MuxinFeng)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🐞 修复 Button 有 `href` 时 `disabled` 属性不生效的问题。[#35952](https://github.com/ant-design/ant-design/issues/35952)。[#35975](https://github.com/ant-design/ant-design/pull/35975) [@MuxinFeng](https://github.com/MuxinFeng)
```

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,23 @@ timeline: true
 
 ---
 
+## 4.21.1
+
+`2022-06-13`
+
+- 🐞 修复 Image 中的 `getContainer` 属性没有读取 ConfigProvider 中的设置。[#36002](https://github.com/ant-design/ant-design/pull/36002) [@robothot](https://github.com/robothot)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:22]:
```suggestion
- 🐞 修复 Image `getContainer` 属性没有从 ConfigProvider 中读取的问题。[#36002](https://github.com/ant-design/ant-design/pull/36002) [@robothot](https://github.com/robothot)
```

**Review by xrkffgg — APPROVED** [review]

**Review by afc163 — APPROVED** [review]

**Pull request merged**